resources/sdk/purecloudjavascript/templates/ApiClient.mustache

```diff
@@ -1,4 +1,5 @@
 import superagent from 'superagent';
+import Configuration from './configuration.js';
 
 /**
  * @module {{#invokerPackage}}{{invokerPackage}}/{{/invokerPackage}}ApiClient
```

Comment:
While you're here could you please replace all "new Buffer" calls to "Buffer.from". It'll remove a depreciation message.

Reply:
Done